src/frontend/typer/manuell-journalføring.ts

```diff
@@ -65,3 +65,55 @@ export interface INavnOgIdent {
     navn: string;
     id: string;
 }
+
+export enum JournalpostTittel {
```

Comment:
I have a bad feeling about using enums like this. Could/should we split the enum into a pure enum and a record for getting the display text?

Reply:
Yes, unfortunately I think these kind of enums often may lead to bugs comparing the value (lower case) when our intention is using the enum name :/  When using uppercase values corresponding to enum name we can be sure of what we get, and can use this are typed key in record like this :D we haven't converted to record all of the places in the app yet, so might be a bit confusing!!

`
export enum BehandlingÅrsak {
    SØKNAD = 'SØKNAD',
    FØDSELSHENDELSE = 'FØDSELSHENDELSE',
    ÅRLIG_KONTROLL = 'ÅRLIG_KONTROLL',
    DØDSFALL = 'DØDSFALL',
    NYE_OPPLYSNINGER = 'NYE_OPPLYSNINGER',
    TEKNISK_OPPHØR = 'TEKNISK_OPPHØR',
}

export const behandlingÅrsak: Record<BehandlingÅrsak, string> = {
    SØKNAD: 'Søknad',
    FØDSELSHENDELSE: 'Fødselshendelse',
    ÅRLIG_KONTROLL: 'Årlig kontroll',
    DØDSFALL: 'Dødsfall',
    NYE_OPPLYSNINGER: 'Nye opplysninger',
    TEKNISK_OPPHØR: 'Teknisk opphør',
};
`